index.js

```diff
@@ -61,32 +56,19 @@ module.exports = class Combobo {
       });
     }
 
+    attrs(this.input, this.list, this.cachedOpts);
+
     // initial state
     this.isOpen = false;
-    this.liveRegion = null;
     this.currentOption = null;
     this.selected = [];
     this.isHovering = false;
 
-    this.initAttrs();
-    this.initEvents();
-  }
-
-  initAttrs() {
```

Comment:
this and setOptionAttrs were only being called once so I moved them to lib (not needed as a method)